### ISSUE TITLE
40-redhat.rules file change causes postscript cuda_power9_setup to fail

### ISSUE THREAD

**Issue by marksmit-ibm** [xcat2/xcat-core#5852]
With RHEL7.6-alt a change in the systemd rpm in file, $chroot//lib/udev/rules.d/40-redhat.rules
results in postscript cuda_power9_setup
failing to remove the hot memory add line in both the diskful and diskless builds.

> 	sed -i /SUBSYSTEM==\"memory\"/d ${IMG_ROOTIMGDIR}/lib/udev/rules.d/40-redhat.rules

The postscript command above finds the line in 7.5 but does not match the new 7.6-alt
SUBSYSTEM=="memory"   changed to
SUBSYSTEM!="memory"

Additionally I am unsure of the GOTO and LABEL purpose, and thus am not sure just deleting the one row would have the desired effect.

7.5-alt:
```
# Memory hotadd request
SUBSYSTEM=="memory", ACTION=="add", PROGRAM="/bin/uname -p", RESULT!="s390*", ATTR{state}=="offline", ATTR{state}="online"
```

7.6-alt:
```
# Memory hotadd request
SUBSYSTEM!="memory", ACTION!="add", GOTO="memory_hotplug_end"
PROGRAM="/bin/uname -p", RESULT=="s390*", GOTO="memory_hotplug_end"

ENV{.state}="online"
PROGRAM="/bin/systemd-detect-virt", RESULT=="none", ENV{.state}="online_movable"
ATTR{state}=="offline", ATTR{state}="$env{.state}"

LABEL="memory_hotplug_end"
```

- [x] A minor error exists in the documentation.

https://xcat-docs.readthedocs.io/en/latest/advanced/gpu/nvidia/osimage/rhels.html#power9-setup
for Diskful seems to have an error.  In the documentation text, 
" Instead, add this postscript to your compute node postbootscrtips list."
In order to match the example: "chdef p9compute -p postscripts=cuda_power9_setup"
I am guessing the diskfull text should read postscripts in place of postbootscripts.

https://xcat-docs.readthedocs.io/en/latest/guides/admin-guides/references/man5/postscripts.5.html
seems to indicate postscripts would be better than postbootscripts.

**Comment by marksmit-ibm** [comment]
systemctl status nvidia-persistenced
seems to be reading correctly after commenting all the lines in the power9 case for diskful/rhel7.6-alt
```
SUBSYSTEM!="memory", ACTION!="add", GOTO="memory_hotplug_end"
PROGRAM="/bin/uname -p", RESULT=="s390*", GOTO="memory_hotplug_end"

ENV{.state}="online"
PROGRAM="/bin/systemd-detect-virt", RESULT=="none", ENV{.state}="online_movable"
ATTR{state}=="offline", ATTR{state}="$env{.state}"

LABEL="memory_hotplug_end"
```

**Comment by marksmit-ibm** [comment]
Commenting out or Removing the first line was not enough to fix the problem.
However, inserting a line after `# Memory hotadd request`
> GOTO="memory_hotplug_end"
did seem to also provide the correct output from `systemctl status nvidia-persistenced`.

**Comment by hu-weihua** [comment]
Can not fix in 2.14.5. move to 2.14.6

**Comment by whowutwut** [comment]
@marksmit-ibm  I fixed the minor issue you mentioned regarding the documentation directly without a pull request using commit: https://github.com/xcat2/xcat-core/commit/9d36c4d25e363c5019d81cbc56f60158b6f0a638

I am looking at the other issue you bring up....

**Comment by whowutwut** [comment]
This issues was resolved by #5947 as the script should be generalized for 7.5 and 7.6.
